src/manifest.ts

```diff
@@ -444,6 +445,9 @@ export class Manifest {
       parseConfig(github, configFile, changesBranch, path, releaseAs),
       parseReleasedVersions(github, manifestFile, changesBranch),
     ]);
+    if (manifestOptions.logger) {
```

Comment:
What does this do?

Reply:
`manifestOptions.logger` is an optional instance to use for logging. By default release-please relies on a gloal logger defined in `src/util/logger.ts`.

When CLI commands are run, `setupLogger()` is called to do some configuration.

In our case we are using release-please via `fromManifest()`, and pass the logger via the option.  But I'm not convinced release-please correctly use the logger from the options all the time. It is safer to also set the global one.